README.md

```diff
@@ -1,11 +1,26 @@
-<img src="https://github.com/fika-lang/assets/blob/master/logo.png?raw=true" width="150"/>
+<img src="https://github.com/fika-lang/assets/blob/master/logo.png?raw=true" width="150"/> &nbsp; ![](https://img.shields.io/github/forks/fika-lang/fika?style=social) ![](https://img.shields.io/github/stars/fika-lang/fika?style=social) ![](https://img.shields.io/github/watchers/fika-lang/fika?style=social) <br>
+
+![](https://img.shields.io/github/repo-size/fika-lang/fika) ![](https://img.shields.io/github/license/fika-lang/fika?color=red)<br>
+![](https://img.shields.io/github/issues/fika-lang/fika?color=green) ![](https://img.shields.io/github/issues-pr/fika-lang/fika?color=green) ![](https://img.shields.io/github/downloads/fika-lang/fika/total) ![](https://img.shields.io/github/last-commit/fika-lang/fika) ![](https://img.shields.io/github/contributors/fika-lang/fika)
+
+---
 
 Fika is a modern programming language for the web.
 It is statically typed, functional and runs on the BEAM (Erlang VM).
 Fika is designed for building and maintaining scalable web apps without
 compromising on programmer ergonomics.
 
-Project status: Actively developed. Not ready for production use or HackerNews.
+**Project status**: Actively developed. Not ready for production use or HackerNews.
+
+### Clone git repository
+
+```sh
+    $ Git clone "https://github.com/fika-lang/fika"
+```
+
+You can run and edit the algorithms or contribute to them using [Gitpod.io](https://www.gitpod.io/), a free online development environment, with a single click.
```

Comment:
> You can run and edit the algorithms or contribute to them using

How about "You can contribute to the repository using ..."

We have this cloning process briefly mentioned by Emil, under "Running Fika programs" section. Can we tidy that up?